tests/test-har.js

```diff
@@ -147,7 +147,7 @@ tape('query', function (t) {
     var json = JSON.parse(body)
 
     t.equal(err, null)
-    t.equal(json.url, '/?fff=sss&foo[0]=bar&foo[1]=baz&baz=abc')
+    t.equal(json.url, '/?fff=sss&foo%5B0%5D=bar&foo%5B1%5D=baz&baz=abc')
```

Comment:
This is the falsy test,
qs will encode the string and gives to url parser, where we decodes it again and sent as [ and ]  which was a bug, fixed now hence it is failing, now the test is fixed along with the issue.
Also,
Added the test in the url-parse to make sure [  and ] are not encoded by us.